evaluation/user_evaluations.go

```diff
@@ -69,3 +69,14 @@ func sortMapKeys(data map[string]string) []string {
 	sort.Strings(keys)
 	return keys
 }
+
+func GenerateFeaturesID(features []*ftproto.Feature) string {
```

Comment:
I'm using the same logic used to generate the [UserEvaluationsID](https://github.com/bucketeer-io/bucketeer/blob/main/evaluation/user_evaluations.go#L47).